### PR TITLE
Various cleanups

### PR DESCRIPTION
**Pull request by dmerejkowsky**
Review: commit by commit (except the last one)

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "tbump"
-version = "6.1.1"
+version = "1.2.42"
```
Comment (maximerety) [inline pyproject.toml:6]: ?

File: tbump/main.py [hunk 2]
```diff
@@ -19,7 +19,7 @@
 from tbump.hooks import HooksRunner
 
 
-TBUMP_VERSION = "6.1.1"
```
Comment (maximerety) [inline tbump/main.py:22]: If changing the versioning, it might deserve its own commit... or is it an accidental change?

Comment (dmerejkowsky) [reply tbump/main.py:22]: It is accidental - there was a buggy test involved. Good catch

**Review by maximerety — APPROVED** [review]

**Review by maximerety — APPROVED** [review]
